modules/admin_manual/pages/qnap/index.adoc

```diff
@@ -0,0 +1,274 @@
+
+. Guest users must be able to access ownCloud either from within the same local network or via the internet.
+
+=== Managing Users
+
+In the menu:admin[Users] section you can create, disable, enable or delete users, add them to groups, grant administrator privileges and more.
+
+For detailed information see xref:configuration/user/user_configuration.adoc[User Management].
+
+After you have created users, they can upload files and share these with other ownCloud users and guest users. Folders can be created and shared as well. Sharing works via the email address of the user who should be granted access to certain files.
+
+Once a regular user shares a file or folder specifying an external email address, the recipient of the email becomes a guest user.
+
+== Enabling ownCloud Apps
+
+With the free community edition, you can use all apps that are published under the GPL or similar public licenses, e.g. the GNU Affero General Public License. These apps you can simply enable and enjoy.
+In the upper left corner, click on the main menu with the three bars, select menu:Market and install what you like.
+
+A bundle of Enterprise Apps is published under the ownCloud Commercial License and only available with the Enterprise Edition. To see what's not included in the free community edition, select `App Bundles` in the left-hand navigation bar.
+
+== Enterprise Edition and Licensing
+
+If you want to use enterprise features, obtain a license from the QNAP Store and activate it in the QNAP LicenseCenter. In case you want to have more regular users, you can buy additional licenses.
+
+Licenses are valid for a year. If you don't renew them, only the first five users created will remain enabled and only non-enterprise apps. Should you decide to buy licenses again at a later point in time, you can enable the users you want to become active again in the web interface.
+
+Users can also be enabled or disabled via `occ` commands. For more information on the ownCloud command line interface, see below.
+
+== Accessing your QNAP ownCloud from the Internet
+
+If you want to connect to your ownCloud on QNAP from the Internet, you need to configure the network accordingly. In the menu:Main menu of your QNAP NAS, select menu:SYSTEMS[Network & Virtual Switch]. Under "Access Services" click on menu:DDNS (Dynamic Domain Name Service) then btn[Add]. Here you can configure the DDNS settings.
+
+// For more information see the official QNAP documentation at: ???
+
+In case you want to assign a static IP address, you need to access your QNAP device via `ssh` on the command line and edit the file `custom/user.config.php` in your top-level ownCloud app directory, e.g. `/share/CACHEDEV1_DATA/.qpkg/ownCloud`.
+Create an entry like in the following example with the correct IP address:
+
+[source,console]
+----
+<?php
+$CONFIG = array(
+    'overwriteprotocol' => 'https',
+    'overwritehost' => '10.20.3.400',
+    'overwrite.cli.url' => 'https://10.20.3.400/owncloud/',
+);
+----
```

Comment:
From the text I guess that ownCloud uses apache to be accessed via a subpage (directory) in the url, but this documentatin does not tell about. Needs to be clarified.

This is important, because if you want to access oC from the outside, you need to tell the `overwrite.cli.url` the correct address, which is either as in the example a subpage but can also be subdomain (https://owncloud/10.20.3.400).

`overwriteprotocol` and `overwritehost` are only important if using a proxy, see https://doc.owncloud.com/server/admin_manual/configuration/server/config_sample_php_parameters.html#proxy-configurations

Reply:
Thanks for the tip. With the next push, I'll add a hint to adjust the URL accordingly if a subdomain is used.